src/components/lime-ai-avatar/examples/lime-ai-avatar-colors.tsx

```diff
@@ -17,19 +17,20 @@ export class LimeAiAvatarColorsExample {
     private backgroundColor = 'rgb(var(--color-blue-default))';
 
     public render() {
-        return [
-            <div class="avatar-container">
-                <limel-lime-ai-avatar
-                    style={{
+        return (
+            <Host>
```

Comment:
you don't like the commas? 😊

Reply:
SonarQube complains about missing `key` properties when we use arrays, so please stop using arrays just to wrap elements 🙏

Reply:
I see. OK